Codes on Turtle Graphics/README.md

```diff
@@ -0,0 +1,20 @@
+# Codes on Turtle Graphics
+
+_turtle_ is a pre installed library in Python that enables us to create pictures and shapes by providing them with a virtual canvas. The onscreen pen that we use to draw
+is called turtle.
+
+### Requirements
+#### 1. Python Environment:
+Use applications like Python IDLE or Jupyter Notebook.
+#### 2. Python Version:
+Ensure you have version 3 of Python.
+
+With the turtle library of Python, we can draw several shapes and figures. Here's two samples we can create with turtles's library.
+
+### 1. Animated Robot
+This is a simple figure we can create to understand basics of turtle graphics.  
+<img src="Robot.png" width="350" height="250">
+
+### 2. Kaleido Spiral
+In this, we will draw circles and squares in continuation. Everytime we will change the radius of circle and side dimension of square to make it look like a expanding squares and circles.  
+<img src="Kaleido-spiral.png" width="350" height="250">
```

Comment:
Images not rendered: 

![image](https://user-images.githubusercontent.com/43691873/112938941-410d5e00-9148-11eb-945b-4eeadd57addf.png)